detection_rules/schemas/definitions.py

```diff
@@ -35,6 +35,7 @@
 MACHINE_LEARNING = 'machine_learning'
 SAVED_QUERY = 'saved_query'
 QUERY = 'query'
+BUILDING_BLOCK_TYPE = 'default'
```

Comment:
Is this the only value possible (from Kibana schema)? More of a curiosity than a blocker

Reply:
```suggestion
```